lib/person.rb

```diff
@@ -0,0 +1,30 @@
+require 'yaml'
+require 'date'
+require './lib/library.rb'
+
+def books
+  YAML.load_file('./lib/books.yml')
+end
+
+class Person
+  attr_accessor :name, :book
+
+  def initialize(name)
+    @name = name
+    @book = []
+  end
+
+  def set_renter(obj)
+    [:renter] == @name
+  end
+
+  def search_my_books
+    books.select { |obj| obj[:item][:renter]}
+    return :return_date
+  end
```

Comment:
If you are searching through your own books, you should do that on the `@book` attribute, that I believe should hold a collection of books you have checked out, right? (apart from the naming issue I mentioned earlier). 

Also, I cant make out what you want to return. The way it is written now `:renter` will always just be a symbol.